src/processes.py

```diff
@@ -0,0 +1,430 @@
+    evaluation_phase(stamped_output_dir, picked_learning_profiles)
+
+    print("Evaluated successfully!")
+
+    # Done
+    _phase_header("DONE")
+
+
+def _get_sorted_specific_learning_profiles(lps, eval_mode, pres_mode,
+                                           n_models_per_category_item):
+    sorted_lps = []
+    # Get all specific learning profiles
+    for spec_lps in get_specific_learning_profiles(
+            lps, pres_mode):
+        # For each attribute, sort learning profiles by score
+        # and choose n_models_per_category_item nr of models per category item
+        for lp in sort_by_score(spec_lps, eval_mode,
+                                n_models_per_category_item):
+            sorted_lps.append(lp)
+    return sorted_lps
+
+
+def presentation_process(learning_profile_dir: Path, n_models: int):
+    """
+    Runs the model evaluation process.
+
+    Args:
+        learning_profile_dir (Path): A directory with Learning Profile results
+            from either model_selection or model_evaluation.
+        n_models (int): Number of models to be included in plots
+            (-1 all models included).
+    """
+    # get profiles
+    lps_desc = load_all_learning_profiles(learning_profile_dir)
+    # Copy
+    lps_present = lps_desc
+
+    ######################
+    # Presentation Phase #
+    ######################
+    _phase_header("PRESENTATION PHASE")
+    print(f"In total there is {len(lps_desc)} Learning profiles.")
+
+    # Setup variables
+    quit = False
+    picked_eval = None
+    eval_modes = [eval_mode for eval_mode in EvaluationMode]
+    pres_modes = [pres_mode for pres_mode in PresentationMode]
+
+    # Input loop to gather plot settings
+    while True:
+        _phase_header("PLOT SETTINGS")
+        ###################
+        # Evaluation mode #
+        ###################
+        # Input prompt
+        print("Pick evaluation mode by writing the index of the desired"
+              " evaluation mode.")
+        for idx, eval_mode in enumerate(EvaluationMode):
+            print(f"{idx}:\t{eval_mode}")
+        print("Write 'exit' to quit.")
+
+        # Handle evaluation mode input
+        while True:
+            try:
+                idx = input("> ")
+                if "exit" == idx:
+                    quit = True
+                    break
+                elif int(idx) >= 0 and int(idx) < len(EvaluationMode):
+                    picked_eval = eval_modes[int(idx)]
+                    break
+            except ValueError:
+                continue
+
+        if quit:
+            break
+
+        #####################
+        # Presentation mode #
+        #####################
+        # Input prompt
+        print("Pick presentation mode by writing the index of the wanted"
+              " presentation mode.")
+        for idx, pres_mode in enumerate(PresentationMode):
+            print(f"{idx}:\t{pres_mode}")
+        print("Write 'all' to present all learning profiles.")
```

Comment:
Write 'all' to present all presentation modes one after the other? 
Should this option exist?